connectors/redis/redis.go

```diff
@@ -0,0 +1,255 @@
+	MaxIdle int
+	// IdleTimeout directs to close connections after remaining idle for this duration.
+	// If the value is zero, then idle connections are not closed. Applications should set
+	// the timeout to a value less than the server's timeout.
+	IdleTimeout time.Duration
+	// Maximum number of connections allocated by the pool at a given time.
+	// When zero, there is no limit on the number of connections in the pool.
+	MaxActive int
+
+	ConnectTimeout time.Duration
+	ReadTimeout    time.Duration
+	WriteTimeout   time.Duration
+}
+
+// NewConnector initializes a Redis Connector
+func NewConnector(config Config) dosa.Connector {
+	return &Connector{
+		client: NewRedigoClient(config.ServerSettings),
+		ttl:    config.TTL,
+	}
+}
+
+// Connector for redis database
+type Connector struct {
+	base.Connector
+	client SimpleRedis
+	ttl    time.Duration
+}
+
+// CreateIfNotExists not implemented
+func (c *Connector) CreateIfNotExists(ctx context.Context, ei *dosa.EntityInfo, values map[string]dosa.FieldValue) error {
+	return new(ErrNotImplemented)
+}
+
+// MultiRead not implemented
+func (c *Connector) MultiRead(ctx context.Context, ei *dosa.EntityInfo, keys []map[string]dosa.FieldValue, minimumFields []string) (results []*dosa.FieldValuesOrError, err error) {
+	return nil, new(ErrNotImplemented)
+}
+
+// MultiUpsert not implemented
+func (c *Connector) MultiUpsert(ctx context.Context, ei *dosa.EntityInfo, multiValues []map[string]dosa.FieldValue) (result []error, err error) {
+	return nil, new(ErrNotImplemented)
+}
+
+// RemoveRange not implemented
+func (c *Connector) RemoveRange(ctx context.Context, ei *dosa.EntityInfo, columnConditions map[string][]*dosa.Condition) error {
+	return new(ErrNotImplemented)
+}
+
+// MultiRemove not implemented
+func (c *Connector) MultiRemove(ctx context.Context, ei *dosa.EntityInfo, multiKeys []map[string]dosa.FieldValue) (result []error, err error) {
+	return nil, new(ErrNotImplemented)
+}
+
+// Range not implemented.
+func (c *Connector) Range(ctx context.Context, ei *dosa.EntityInfo, columnConditions map[string][]*dosa.Condition, minimumFields []string, token string, limit int) ([]map[string]dosa.FieldValue, string, error) {
+	return nil, "", new(ErrNotImplemented)
+}
+
+// Search not implemented.
+func (c *Connector) Search(ctx context.Context, ei *dosa.EntityInfo, fieldPairs dosa.FieldNameValuePair, minimumFields []string, token string, limit int) (multiValues []map[string]dosa.FieldValue, nextToken string, err error) {
+	return nil, "", new(ErrNotImplemented)
+}
+
+// Scan not implemented.
+func (c *Connector) Scan(ctx context.Context, ei *dosa.EntityInfo, minimumFields []string, token string, limit int) (multiValues []map[string]dosa.FieldValue, nextToken string, err error) {
+	return nil, "", new(ErrNotImplemented)
+}
+
+// CheckSchema not implemented.
+func (c *Connector) CheckSchema(ctx context.Context, scope string, namePrefix string, ed []*dosa.EntityDefinition) (version int32, err error) {
```

Comment:
remove any of the schema operations; this will cause the requests to go to the base connector, which returns a different error (NoMoreConnectors). That's more appropriate for these schema errors.